### PR TITLE
feat: insert real data into db

### PR DESCRIPTION
**Pull request by will249**

### CODE REVIEW

File: src/utils/populateDatabase.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const entries: EmbeddingDTO[] = JSON.parse(
+  fs.readFileSync('./src/utils/data/dummyEmbeddings.json', 'utf8')
+);
+
+const supabase = createClient(
+  process.env.NEXT_PUBLIC_SUPABASE_URL ?? '',
+  process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY ?? ''
+);
+
+const insertItems = async (
```
Comment (matt-newhall) [inline src/utils/populateDatabase.ts:21]: could call this `populateDatabase` to match file name (or rename file)

File: src/utils/populateDatabase.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import dotenv from 'dotenv';
+import * as path from 'path';
+import * as fs from 'fs';
+import { AppSupabaseClient, Table } from '@/types';
+import { createClient } from '@supabase/supabase-js';
+import { Database } from '../lib/database.types';
+
+dotenv.config({ path: path.resolve(__dirname, '../..', '.env.local') });
+
+type EmbeddingDTO = Database['public']['Tables']['embedding']['Insert'];
```
[the remark this answers is not in the document]
Comment (will249) [reply src/utils/populateDatabase.ts:10]: Yes i see your point, but as it's already defined in database.types (albeit nested) is there any point in defining elsewhere for reuse?

File: src/utils/populateDatabase.ts (hunk 1 above)
Comment (will249) [reply src/utils/populateDatabase.ts:21]: true true

File: src/utils/populateDatabase.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (matt-newhall) [reply src/utils/populateDatabase.ts:10]: i mean as it's defined in two places would it not be easier to define it once and import it in the other?

[the remark this answers is not in the document]
Comment (will249) [reply src/utils/populateDatabase.ts:10]: ah ok true, will amend

**Review by matt-newhall — APPROVED** [review]

**Pull request merged**